app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -84,4 +110,18 @@ def clear_session_data
     user_session.delete(unconfirmed_phone_session_key)
     user_session.delete(confirmation_code_session_key)
   end
+
+  def sms_enabled?
```

Comment:
Yes, the OtpDeliveryFallback is used in two_factor_auth_controller and the PhoneConfirmationFlow is used in phone_confirmation_controller. I would have to make another concern to remove that method, not worth it at this point.